mockOSApi.v2/Data/DTO/ProcessDto.cs

```diff
@@ -0,0 +1,78 @@
+using mockOSApi.Models;
+using mockOSApi.Repository;
+
+namespace mockOSApi.DTO;
+
+public class PriorityClass()
+{
+    public int Prio { get; set; }
+    public int Pid { get; set; }
+}
+
+public class ProcessDto()
+{
+    public int Pid { get; set; }
+    public string? Image { get; set; }
+
+    public PROC_STATE State { get; set; }
+
+    public int Priority { get; set; }
+
+}
+
+public class ProcessCreationDto()
+{
+
+    public int Pid { get; set; }
+    public string? Image { get; set; }
+
+    public string[]? Arguments { get; set; }
+
+    public int Priority { get; set; }
+
+}
+
+
+public interface IMapper
```

Comment:
This should be in a different file.

I imagine you started to find this mapping logic tedious.
Let me introduce you to [Automapper](https://docs.automapper.org/en/stable/Getting-started.html)

Reply:
Have removed manually created mappers and used Automapper instead, thank for tip ;)